include/RAJA/policy/cuda/intrinsics.hpp

```diff
@@ -23,7 +23,7 @@
 
 #include "RAJA/config.hpp"
 
-#if defined(RAJA_ENABLE_CUDA)
+#if defined(RAJA_CUDA_ACTIVE)
```

Comment:
What's the difference between these macros?

Reply:
I believe `RAJA_ENABLE_CUDA` is true when CUDA is enabled in the build and that `RAJA_CUDA_ACTIVE` is true when a code region is being compiled for the device. Somebody, please check me on this.

Reply:
Yep, its about allowing you to use the same RAJA that was configured with gpus in both files compiled as cuda/hip and files compiled as cpp (without a gpu compiler).